src/greet/greet.test.ts

```diff
@@ -1,7 +1,7 @@
 import greet from './greet';
 
-describe('greet', (): void => {
```

Comment:
Explicit return type are not needed because we deactivated the rule for that.

Reply:
👍